test/external/external_test.go

```diff
@@ -19,6 +19,15 @@ import (
 	. "github.com/onsi/gomega/gbytes"
 )
 
+func saveCacheFolder() string {
+	return filepath.Join(os.TempDir(), "relok8s-save-cache")
```

Comment:
This is actually deleting the cache that's being used by relok8s itself right? I don't like that running a test has the side-effect of deleting the actual tooling cache.

Could we instead run these tests pointing to an unique, per-test cache path? This might require adding support for our tool via ENV variable or flag to point to the cache directory.

Reply:
Do not want to add more code to be able to test, we already abandoned a change for that.

Maybe I can remove the specific layer instead.

Reply:
nah, don't worry about it

Reply:
Done